src/main/java/de/dataelementhub/rest/controller/v1/ExportController.java

```diff
@@ -0,0 +1,96 @@
+package de.dataelementhub.rest.controller.v1;
+
+import de.dataelementhub.model.dto.export.ExportInfo;
+import de.dataelementhub.model.dto.export.ExportRequest;
+import de.dataelementhub.model.service.ExportService;
+import de.dataelementhub.rest.DataElementHubRestApplication;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1")
+public class ExportController {
+
+  private final ExportService exportService;
+
+  @Autowired
+  public ExportController(ExportService exportService) {
+    this.exportService = exportService;
+  }
+
+  public static String exportDirectory = System.getProperty("user.dir") + "/uploads/export/";
+
+  /**
+   * Return an overview of all exports.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @GetMapping(value = "/export")
+  public ResponseEntity<List<ExportInfo>> allExports() {
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    List<ExportInfo> exportDescriptions = exportService.allExports(userId, exportDirectory);
+    return new ResponseEntity<List<ExportInfo>>(exportDescriptions, HttpStatus.ACCEPTED);
+  }
+
+  /**
+   * Start an export process for all given elements.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @PostMapping(value = "/export")
+  public ResponseEntity<String> export(@RequestBody ExportRequest exportRequest,
+      @RequestParam String format, @RequestParam Boolean fullExport) {
+    int numberOfNamespacesExportedFrom = exportRequest.getElementUrns().stream()
+        .map(e -> e.split(":")[1]).collect(Collectors.toSet()).size();
+    if (numberOfNamespacesExportedFrom > 1) {
+      return new ResponseEntity<>("Export from more than one namespace is forbidden",
+          HttpStatus.NOT_ACCEPTABLE);
```

Comment:
Not Acceptable would mean that the client provided a list of acceptable content in its Accept header and none of those can be supplied. (See [RFC](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.6)).
More appropriate would be to send back a simple 400 Bad request since the request sent by the client is not acceptable.